File: DevExpress-Demo/CS/MVVMDemo.Wpf/Modules/BindableBaseModule.cs
using MVVMDemo.BindableBaseDemo;
using DevExpress.Xpf.DemoBase;

namespace MVVMDemo {
    public partial class BindableBaseModule : MVVMDemoModule {
        protected override IShowcaseInfo[] CreateShowcases() {
            const string path = "Modules/BindableBase";
            const string uri = "17350/Common-Concepts/MVVM-Framework/ViewModels/BindableBase";
            return new ShowcaseInfo[] {
                LoadShowcase("Bindable Properties", uri, path, new[] { typeof(BindablePropertiesViewModel), typeof(BindablePropertiesView) }),
                LoadShowcase("Bindable Properties via Fields", null, path, new[] { typeof(BindablePropertiesViaFieldsViewModel), typeof(BindablePropertiesViaFieldsView) }),
            };
        }
    }
}

File: DevExpress-Demo/VB/MVVMDemo.Wpf/Modules/BindableBaseModule.vb
Imports MVVMDemo.BindableBaseDemo
Imports DevExpress.Xpf.DemoBase

Namespace MVVMDemo

    Public Partial Class BindableBaseModule
        Inherits MVVMDemoModule

        Protected Overrides Function CreateShowcases() As IShowcaseInfo()
            Const path As String = "Modules/BindableBase"
            Const uri As String = "17350/Common-Concepts/MVVM-Framework/ViewModels/BindableBase"
            Return New ShowcaseInfo() {LoadShowcase("Bindable Properties", uri, path, {GetType(BindablePropertiesViewModel), GetType(BindablePropertiesView)}), LoadShowcase("Bindable Properties via Fields", Nothing, path, {GetType(BindablePropertiesViaFieldsViewModel), GetType(BindablePropertiesViaFieldsView)})}
        End Function
    End Class
End Namespace
